packages/app/app/containers/ListeningHistoryContainer/ListeningHistoryContainer.test.tsx

```diff
@@ -40,15 +40,25 @@ describe('Listening history container', () => {
   it('renders the listening history', async () => {
     const { component } = mountComponent();
 
+    const makeLocal = (dateString, timeString?) => {
+      const date = new Date(dateString).toLocaleDateString();
```

Comment:
Can you do the same thing with `timeString` for consistency?

Reply:
Sorry, I meant just assigning the time to a const, like you did with date. I can't see the previous version because somehow there's still only one commit.

Reply:
Ahhh ok, sure no problem will make that change. Sorry,  I squashed the previous commit down with a rebase.